From d8000e0bf432de5c7bb8454e660c6d6d75f7df0c Mon Sep 17 00:00:00 2001
From: Vinod Koul <vinod.koul@intel.com>
Date: Tue, 9 Aug 2011 10:08:10 +0530
Subject: dmaengine: remove struct scatterlist for header

Commit 90b44f8 introduces dmaengine_prep_slave_single API which adds
scatterlist.h in dmaengine.h, so defining struct scatterlist is not required

Signed-off-by: Vinod Koul <vinod.koul@intel.com>
Acked-by: Dan Williams <dan.j.williams@intel.com>
(cherry picked from commit a16e470caa173d323ef68dcac98c899b95fa4f84)

Conflicts:

	include/linux/dmaengine.h

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 include/linux/dmaengine.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/linux/dmaengine.h b/include/linux/dmaengine.h
index 2ae9e01..0d079a4 100644
--- a/include/linux/dmaengine.h
+++ b/include/linux/dmaengine.h
@@ -26,8 +26,6 @@
 #include <linux/dma-mapping.h>
 #include <linux/scatterlist.h>
 
-struct scatterlist;
-
 /**
  * typedef dma_cookie_t - an opaque DMA cookie
  *
-- 
1.7.10.2.565.gbd578b5

